pyproject.toml

```diff
@@ -1,9 +1,16 @@
 [tool.black]
-line-length = 88
+line-length = 99
```

Comment:
pep8 actually has a note that 99 is also acceptable, and I think works better for larger screens, and less of a nusance for us to change to, so considering this a compromise